src/evaluate_fit.jl

```diff
@@ -43,9 +43,13 @@ function col_objective(glrm::AbstractGLRM, j::Int, y::AbstractArray, X::Array{Fl
     if length(sz) == 1 colind = 1 else colind = 1:sz[2] end
     err = 0.0
     XY = X'*y
-    for i in glrm.observed_examples[j]
+    obsex = glrm.observed_examples[j]
+    @inbounds XYj = XY[obsex,colind]
+    @inbounds Aj = convert(Array, glrm.A[obsex,j])
```

Comment:
The conversion is necessary because A might be a DataFrame or a DataMatrix (2-dimensional DataArray), but the hardware doesn't like those nearly as much as it likes native arrays.